### ISSUE TITLE
Error fixing SageMath 9.2

### ISSUE THREAD

**Issue by jjtorrens** [3-manifolds/fix_mac_sage#1]
I want to fix an installation of SageMath 9.2. To this end, I download `fix_mac_sage9tar.gz`, uncompress it and move the `fix_mac_sage9` folder to the desktop. Since the Sage program is at `/Applications/SageMath`, the README file says that I should run the following commands:

    $ cd ~/Desktop/fix_mac_sage9
    $ /Applications/SageMath/sage -python fix_mac_sage9.fix

However, there is no file `fix_mac_sage9.fix`, but `fix.py`. I try instead

    $ /Applications/SageMath/sage -python fix.py

But this triggers the error

    Traceback (most recent call last):
      File "fix.py", line 12, in <module>
        from . import __path__
    ImportError: attempted relative import with no known parent package

What should I do to achieve my goal? Many thanks

**Comment by NathanDunfield** [comment]
Since, in your case,  the `fix_mac_sage9` directory is in `~/Desktop`, you want to do:
```
$ cd ~/Desktop
$ /Applications/SageMath.app/sage -python -m fix_mac_sage9.fix
```
Note the `-m` in the last line.  Does that fix your problem?

**Comment by jjtorrens** [comment]
I had misinterpreted the instructions in the README file. However, the problem continues. Since SageMath 9.2 comes with Python 3.8, I have changed `python3.7` to `python3.8` in `fix.py` and  renamed the folder in `fix_mac_sage9/local/lib`. This allows your code to run without errors. But when I launch the Jupyter notebook as usual (`sage -n jupyter`), I get these error messages:

```
Please wait while the Sage Jupyter Notebook server starts...
Traceback (most recent call last):
  File "/Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/site-packages/sage/repl/ipython_kernel/install.py", line 307, in have_prerequisites
    from notebook.notebookapp import NotebookApp
  File "/Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/site-packages/notebook/notebookapp.py", line 66, in <module>
    from tornado import httpserver
  File "/Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/site-packages/tornado/httpserver.py", line 29, in <module>
    import ssl
  File "/Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/ssl.py", line 98, in <module>
    import _ssl             # if we can't import it, let the error propagate
ImportError: dlopen(/Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/lib-dynload/_ssl.so, 2): Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/lib-dynload/_ssl.so (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Applications/SageMath-9.2.app/Contents/Resources/sage/local/lib/python3.8/lib-dynload/_ssl.so
```

My Mac runs macOS 10.13.6. Perhaps this is part of the problem. I have tried the SageMath install provided by the`.dmg`download as well as the `-app.dmg` download, getting the same errors.

**Comment by NathanDunfield** [comment]
I think the problem is most likely that SageMath 9.2 uses Python 3.8, not Python 3.7.  While changing the script allowed it to install, binary Python extension modules (here `_ssl.so`) built for one version of Python cannot be used with another.  

So likely switching to SageMath 9.1 would allow our fix to work.  If you want to stick with 9.2, another option would be to use [this alternate way of installing Sage on macOS](https://doc.sagemath.org/html/en/installation/conda.html).

**Comment by culler** [comment]
It sounds like I need to put together a new fixit package for Sage 9.2.  I
will do that.

- Marc

**Comment by culler** [comment]
I have added a new release which fixes Sage 9.2.

**Comment by NathanDunfield** [comment]
Marc: Thanks, I also updated the install instructions to including removing the quarantine xattr.  Closing issue!
